incubator/knative/templates/build/serviceaccount.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and (.Values.build.install) (not .Values.serving.install ) -}}
+{{ if .Values.serviceAccount.create }}
```

Comment:
.Values.serviceAccount.create does not exist - it seems to be enabled per item - so the conditional needs to follow that

Reply:
This has been fixed.